tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
     py{27,35,36}-django-111
```

Comment:
According to https://docs.djangoproject.com/en/1.11/faq/install/#what-python-version-can-i-use-with-django, Django 1.11 should run with Python 3.4 too. Could you please add it too?

Reply:
I think you both got there at the same time!